app/lib/katello/util/path_with_substitutions.rb

```diff
@@ -1,6 +1,8 @@
 module Katello
   module Util
     class PathWithSubstitutions
+      ARCHITECTURES = ["x86_64", "s390x", "ppc64le", "aarch64" ].freeze
```

Comment:
I would add 'multiarch' and 'ppc64' here as well to cover the container and rhel7 cases